libethereum/BlockChain.cpp

```diff
@@ -941,6 +941,7 @@ ImportRoute BlockChain::import(VerifiedBlockRef const& _block, OverlayDB const&
 	{
 		cnote << "SLOW IMPORT:" << _block.info.hash() << " #" << _block.info.number();
 		cnote << "  Import took:" << total.elapsed();
+		cnote << "  gas per second:" << static_cast<double>(_block.info.gasUsed()) / enactment;
```

Comment:
I guess it's fine if we loose some precision here if gas is a large value?

(if it's a problem we could cast to boost::multiprecision::cpp_dec_float_100)